packages/xstate-scxml/package.json

```diff
@@ -34,7 +34,7 @@
   },
   "dependencies": {
     "xml-js": "^1.6.11",
-    "xstate": "next"
```

Comment:
`next` has caused it to be fetched from npm (instead of using local monorepo version)